README.md

```diff
@@ -34,6 +34,31 @@ yarn && yarn run start
 If you now navigate to [localhost:8080](https://localhost:8080), you will see a list of examples
 that can be explored in the browser.
 
+### Building on Macbook M1
+
+Building Reveal on Macbook M1 migth require some special care.
+
+If you experience issues during the `yarn`-stage in `viewer/`, e.g.
+```
+   no template named 'remove_cv_t' in namespace 'std'
+```
+try forcing GCC to build for C++14:
+
+`env CXXFLAGS=-std=c++14 yarn`
+
+For solving issue with puppeteer in `examples/` you should follow additional steps:
+
+1. Install chromium from Homebrew: 
+```
+brew install chromium
+`which chromium`
+```
+
+2. Inside `examples/` add two environment variables from .env file by: 
+```
+source .env
+```
```

Comment:
Is it possible to have this automated during build somehow?

Reply:
Probably yes, if yarn can check that chromium is installed before downloading packages and then run this command